src/scss/_icons.scss

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
This file should be place in the  components/icons folder.

Reply:
I agree. And name of the file should be without underscore.